common/locales/en/settings.json

```diff
@@ -90,8 +90,8 @@
     "passwordSuccess": "Password successfully changed",
     "usernameSuccess": "Login Name successfully changed",
     "emailSuccess": "Email successfully changed",
-    "detachFacebook": "De-register Facebook",
-    "detachedFacebook": "Successfully removed Facebook from your account",
+    "detachSocial": "De-register <%= network %>",
+    "detachedSocial": "Successfully removed authentication method from your account",
```

Comment:
This is passed as an argument to the `http` function. ( `$http("delete", ApiUrl.get() + "/api/v3/user/auth/social/"+networkKey, null, "detachedSocial");` ) Is it even possible to pass in the network in this case?